marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-step-runner.json

```diff
@@ -0,0 +1,39 @@
+{
+  "role-name": "hub-central-step-runner",
+  "description": "Permits reading/reviewing mapping steps in Hub Central",
+  "role": [
+    "hub-central-user",
+    "data-hub-common-writer",
+    "data-hub-flow-reader",
+    "data-hub-ingestion-reader",
+    "data-hub-mapping-reader",
+    "data-hub-step-definition-reader"
```

Comment:
should include data-hub-match-merge-reader and data-hub-custom-reader per HC roles wiki.

Reply:
created the data-hub-custom-reader role and and added both data-hub-match-merge-reader and data-hub-custom-reader.

Reply:
While the Wiki states that, the Wiki is communicating the end state. Since we don't have referenced custom/match/merge steps in 5.3, I don't think we need to create the roles, nor have them here. It's fine if the work is done already, just pointing out that the Wiki is problematic as a definition of scope when it references many tickets.